projects/optic/src/commands/diff/diff-all.ts

```diff
@@ -79,6 +79,14 @@ comma separated values (e.g. "**/*.yml,**/*.json")'
       '--standard <standard>',
       'run comparison with a locally defined standard, if not set, looks for the standard on the [x-optic-standard] key on the spec, and then the optic.dev.yml file.'
     )
+    .addOption(
+      new Option(
+        '--validation <validation>',
```

Comment:
It's kind of more correct to call it `--head-validation`, however this would be inconsistent with diff since if you run `diff file1 file2` that --validation applies to both